templates/pdashboards.yaml.erb

```diff
@@ -0,0 +1,3 @@
+# This file is managed by Puppet, any changes will be overwritten
+
+<%= @pdashboards.to_yaml %>
```

Comment:
please convert this to an epp template. Also [to_yaml](https://github.com/voxpupuli/puppet-grafana/blob/master/metadata.json#L17) got introduced in version 4.20 in stdlib, please bump the lower version range.